src/libraries/token/VariableSupplyToken.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/libraries/token/StandardToken.sol';
+
+
+contract VariableSupplyToken is StandardToken {
+    using SafeMath for uint256;
+
+    event Mint(address indexed target, uint256 value);
+    event Burn(address indexed target, uint256 value);
+
+    function mint(address _target, uint256 _amount) internal returns (bool) {
```

Comment:
Fixed the returns.  I do agree that we should get some test coverage for this contract, though it is unclear to me how that is related to `test_dispute_bond_token.py`?  This is intended to be used as a base class for some of the internal tokens, but it shouldn't (and doesn't so far) change any external behavior of those tokens.  Its just to simplify the code in a number of places.